docs/website/src/layout/theme.js

```diff
@@ -3,6 +3,7 @@ import lightBlue from '@material-ui/core/colors/lightBlue';
 import {styles} from '@material-ui/core/Typography/Typography';
 
 // A custom theme for this app
+const scrollMarginTop = '74px' // To compencate AppBar on scrolling to headers
```

Comment:
Shouldnt it be `'calc(74px + 1rem)' // Compensate AppBar height + some margin`
And check your spelling in comments, second one.